integration-tests/pom.xml

```diff
@@ -95,33 +95,29 @@
 							<workingDirectory>${sidecar.dir}</workingDirectory>
 						</configuration>
 					</execution>
+				</executions>
+			</plugin>
 
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>exec-maven-plugin</artifactId>
+				<version>3.2.0</version>
+				<executions>
 					<execution>
-						<id>start-sidecar</id>
-						<goals>
-							<goal>npx</goal>
-						</goals>
 						<phase>pre-integration-test</phase>
-						<configuration>
-							<arguments>pm2 -l sidecar.log --time start --name=sidecar npm -- run start</arguments>
-							<skip>${skipTests}</skip>
-							<workingDirectory>${sidecar.dir}</workingDirectory>
-						</configuration>
-					</execution>
-
-					<execution>
-						<id>stop-sidecar</id>
+						<id>start-sidecar</id>
 						<goals>
-							<goal>npx</goal>
+							<goal>exec</goal>
 						</goals>
-						<phase>post-integration-test</phase>
-						<configuration>
-							<arguments>pm2 delete sidecar</arguments>
-							<skip>${skipTests}</skip>
-							<workingDirectory>${sidecar.dir}</workingDirectory>
-						</configuration>
 					</execution>
 				</executions>
+				<configuration>
+					<executable>node</executable>
```

Comment:
Does this work if the node executable is not available in the path? Would it find the node.js installation from the CDS Maven Plugin?

Reply:
You can inject the installation path of the node binary, see here: https://cap.cloud.sap/docs/java/assets/cds-maven-plugin-site/install-node-mojo.html
${cds.node.executable}: Location of the node executable